doximity/orientation
# Improve algolia search

PT Story Link: https://www.pivotaltracker.com/story/show/148938015

QA Testing Instructions:
- visit: https://dox-orientation-qa.herokuapp.com/
- search results should show matched snipped just like the autosuggest does (matched words are bold)
- ensure that search suggestions matches what is in search results, e.g. search for `Neo4j`, the x suggestions that show up in autocomplete should be the top x search results
- archived articles should not appear in autosuggest of other sections (go to archived section, find a title of an article, try looking for it in another section)
- freestyle search and look for any inconsistencies
